cmd/podman/varlink/io.podman.varlink

```diff
@@ -756,15 +756,9 @@ error ContainerNotFound (name: string)
 # PodNotFound means the pod could not be found by the provided name or ID in local storage.
 error PodNotFound (name: string)
 
-# PodNoContainers means the pod does not have any containers associated and cannot be started or restarted.
-error PodNoContainers (name: string)
-
-# PodHasContainers means the pod has containers associated and cannot be stopped without the force boolean set as true.
-error PodHasContainers (name: string)
-
 # PodContainerError means a container associated with a pod failed to preform an operation. It contains
 # a container ID of the container that failed.
-error PodContainerError (names: string)
+error PodContainerError (podname: string, ctrnames: []string)
```

Comment:
Other Errors in the VarLink API carry the "reason" for the error as well as details. The "reason" is used in logging/displaying the issue.

Reply:
fixed:  changed ctrnames to errors, which is a slice of ctr id and reason.